microsoft-edge/webview2/concepts/webview-features-flags.md

```diff
@@ -0,0 +1,144 @@
+---
+title: WebView2 browser flags
+description: Overview of browser flags (arguments, or switches) that affect the WebView2 control, for testing forthcoming features or diagnosing issues.
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.service: microsoft-edge
+ms.subservice: webview
+ms.date: 02/15/2024
+---
+# WebView2 browser flags
+
+One of the ways you can interact with Microsoft Edge WebView2 and impact behavior is by passing browser flags (arguments, or switches) to WebView2.  These browser flags are useful for testing forthcoming features, and for diagnosing issues.  
+
+> [!WARNING]
+> Apps in production shouldn't use WebView2 browser flags, because these flags might be removed or altered at any time, and aren't necessarily supported long-term.
+
+Generally, the flags are owned by both Chromium and Microsoft Edge.  Chromium flags are not owned or controlled by Microsoft Edge, so Microsoft Edge doesn't have control over when or how the flags are altered in their behavior or removed.
```

Comment:
Can we say "when or how the flags are removed or altered in their behavior"?

Reply:
revised